test/parallel/test-microtask-queue-integration-domain.js

```diff
@@ -1,7 +1,16 @@
 'use strict';
 require('../common');
 var assert = require('assert');
-var domain = require('domain');
+
+// Requiring domain calls _setupDomainUse which in turns changes the
```

Comment:
Not very important, but just an observation: I would remove the `calls _setupDomainUse part` and just keep the `changes the function that is used to call callbacks...`. The reason is that that much details is not needed, and it would make the comment depend too much on implementation.